src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/flow/FlowAnnotatorUnitTest.java

```diff
@@ -57,7 +58,7 @@ public Object[][] getTestData() {
                 {
                         // not hmer indel
                         "TATCT CA TTGACCAA", "C",
-                        "del", "1", "1", "A", "ATCTC", "TTGAC", "0.3", "NA", "non-h-indel"
```

Comment:
You can update the comment above this test case since it is now considered an hmer

Reply:
Fixed